### DISCUSSION TITLE
User stories/Questions to be clarified

### DISCUSSION THREAD

**Discussion by Tommy-Sand** [CMPUT404-Big-Yoshi-Social-Network/yoshi-connect#49] [General]
Questions
1. Is a node the same as a server, i.e, is a node controlled by a server admin who can host multiple authors on a node? Reference: " I browse them and I click like on anything by friend Steph who is on another node.

When I click like, my node sends a like object to Steph’s inbox that references her post."

2. Are there instances where an author can modify another authors post. US A.22 US A.23?
3. Should an author have multiple streams (e.g., Public stream, friends only stream, Direct messages)?

User stories to clarify
1. US SA.05 As a server admin, I don’t want to do heavy setup to get the posts of my author’s friends. Clarification needed on what qulifies as heavy setup.
2. US A.11 Pull in my github activity to my "stream". What activity specifically should an author be able to "pull in".

**Comment by kezzayuno** [comment]
1. Yes.
2. Authors can only edit their own posts but admins can edit anyones
3. One stream but how the UI looks does not matter 
1. Administrator want to connect to another node; it should require: putting the URI and authentication of a username and password
2. Activity API on GitHub Documentation
